tests/test_crs/valid_fiod_hostpath.yaml

```diff
@@ -25,7 +25,6 @@ spec:
         - randrw
       bs:
         - 4KiB
-        - 64KiB
       numjobs:
         - 1
```

Comment:
Do we still want numjobs 1 and 2 here or just 2 like in the other fio cr?

Reply:
@dry923  I deliberatelly left numjobs 1 and 2 here for better test coverage because we're not covering it in the other yaml.

Reply:
ack. Sounds good to me.